ScrollingControls/ControlContainableScrollViews.swift

```diff
@@ -41,10 +44,11 @@ final class ControlContainableScrollView: UIScrollView {
 final class ControlContainableTableView: UITableView {
 
     override func touchesShouldCancel(in view: UIView) -> Bool {
-        if view is UIControl
-            && !(view is UITextInput)
-            && !(view is UISlider)
-            && !(view is UISwitch) {
+        if view is ScrollAllowingElement ||
+            (view is UIControl
+                && !(view is UITextInput)
```

Comment:
Instead of manually checking a bunch of types, it would be nice to just conform `UIControl` to `ScrollAllowingElement,` and make a separate protocol for `ScrollDisallowingElement`. Conform `UITextInput`, `UISlider`, and `UISwitch` to that second protocol in this file, and then just check `view is ScrollAllowingElement && !(view is ScrollDisallowingElement)`.